docs/tables/aws_account_contact.md

```diff
@@ -0,0 +1,20 @@
+# Table: aws_account_contact
+
+We can store contact information about the primary account contact for your AWS account.
```

Comment:
This is not a great description, as the table is not storing anything. Please look for a better description